### ISSUE TITLE
Implement waku/0 or /1 - addition of Waku mode to waku protocol

### ISSUE THREAD

**Issue by kdeme** [status-im/nim-eth#124]
Implement a draft of the Waku mode. This could be something similar as https://github.com/status-im/nim-eth/pull/114, but exact spec has to be still decided/written. Draft spec writing might go in parallel of this implementation as PoC (testing ground).

Draft version would probably still fall under waku/0.
Finalized spec + implementation could be waku/1 or waku/0. This has to be still decided and is not part of this issue.

Requirements:
- https://github.com/status-im/nim-eth/issues/123 (or at least the baseline).

Acceptance:
Working PoC of Waku mode.

**Comment by oskarth** [comment]
See https://github.com/vacp2p/specs/pull/61 for draft PR, comments welcome

**Comment by kdeme** [comment]
Basic PoC is there, so testing can be done.

Should eventually be reworked  (code and spec wise), e.g: getting rid of some of the different modes, we probably can get away with two types, e.g. full node and "new" light node.
